kong.conf.default

```diff
@@ -341,6 +341,12 @@
                          # healthiness, and other non-sensitive information
                          # of the current Kong node.
                          #
+                         # Some suffixes can be specified for each pair:
```

Comment:
Since only one suffix is available for this endpoint, can we rephrase this?

Reply:
@hbagdi, I though about it :-), but then something in me said that someone forgets about it will be forgotten when someone adds more, but as you found it right away, that warrants rephrasing :-)